tools/mmp/driver.cs

```diff
@@ -479,7 +479,7 @@ static void Main2 (string [] args)
 			}
 
 			if (!IsUnifiedMobile && tls_provider != null)
-				throw new MonoMacException (2011, true, "Selecting a TLS Provider is only supported in the Unified Mobile profile");
+				throw new MonoMacException (2011, true, "Selecting a TLS Provider is only supported in the Unified Modern (Mobile) profile");
```

Comment:
I opened another PR to remove the check and removed this change so it won't cause a merge conflict with the removal.